### ISSUE TITLE
absence of both posterior communicating arteries

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9615]
**Preferred term label:**
absence of both posterior communicating arteries

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
613686 
PMID: 20087400 2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
